### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,35 +168,27 @@ $ make run
...
-
-## Clean up the deploy
-
-```bash
-$ make destory
+make deploy-work-manager-addon
+make deploy-managed-serviceaccount-addon
 ```
 
 ## Install the multicluster-controlplane and add-ons
```
Comment (clyang82) [inline README.md:190]: ditto

File: README.md [hunk 2]
```diff
@@ -143,35 +168,27 @@ $ make run
...
+clusteradm join --hub-token <controlplane token> --hub-apiserver <controlplane apiserver> --cluster-name <cluster name>
+clusteradm --kubeconfig=<controlplane kubeconfig file> accept --clusters <cluster name>
 ```
 
-> **Warning**
-> clusteradm version should be v0.4.1 or later
+**Note**: clusteradm version should be v0.4.1 or later
 
 
 ## Install add-on
```
[the remark this answers is not in the document]
Comment (skeeey) [reply README.md]: remvoed

File: README.md [hunk 3]
```diff
@@ -41,83 +66,83 @@ Field `etcd` contains config for the controlplane etcd:
...
+      ```
+
+  - To use the node port serive with the following arguements:
+
+      ```bash
+      --set nodeport.enabled=true
+      --set nodeport.port=<your-node-port>
+      ```
+
+  - To enable the self management with the following arguements:
```
[the remark this answers is not in the document]
Comment (skeeey) [reply README.md:128]: yeah, we may add a feature list in readme to address the each feature of controlplane

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**